doc/dev/release-procedure.md

```diff
@@ -346,10 +362,12 @@ Allocator test: fuzz i=0x20
 Allocator test: fuzz i=0x28
 Allocator test: fuzz i=0x30
 Allocator test: fuzz i=0x38
-Test runner: Allocator finished in 30477954 cycles
-Test runner: All tests finished in 42895559 cycles
+Test runner: Allocator finished in 36229517 cycles
+Test runner: All tests finished in 53675674 cycles
```

Comment:
Do we really need to update this every release?

Reply:
Ok, can revert this.